docs/index.md

```diff
@@ -168,64 +199,64 @@ To install `pixi`, run:
     More details on how to use Pixi with Python can be found in the [Python tutorial](python/tutorial.md).
 
 === "Rust"
-    1. **Initialize a workspace:**
-        ```
-        pixi init pixi-rust
-        cd pixi-rust
-        ```
-    2. **Add dependencies:**
-        ```
-        pixi add rust
-        ```
-    3. **Create your project:**
-        ```
-        pixi run cargo init
-        ```
-    4. **Add a task:**
-        ```
-        pixi task add start cargo run
-        ```
-    5. **Run the task:**
-        ```
-        pixi run start
-        ```
-        ```
-        ✨ Pixi task (start): cargo run
-            Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.02s
-             Running `target/debug/pixi-rust`
-        Hello, world!
-        ```
+1. **Initialize a workspace:**
```

Comment:
I assume the formatter changes in this file weren't on purpose. It also breaks the front page